lib/flying_penguin_web/live/search_live/new.html.heex

```diff
@@ -9,19 +9,19 @@
     phx-submit="search">
   
     <%= label f, :origin %>
-    <%= text_input f, :origin, placeholder: "ord" %>
+    <%= text_input f, :origin, value: "ORD" %>
     <%= error_tag f, :origin %>
   
     <%= label f, :destination %>
-    <%= text_input f, :destination, placeholder: "bkk" %>
+    <%= text_input f, :destination, value: "ATL" %>
     <%= error_tag f, :destination %>
   
     <%= label f, :date_of_departure %>
-    <%= date_select f, :date_of_departure, default: DateTime.utc_now() %>
+    <%= date_select f, :date_of_departure, default: DateTime.utc_now() |> DateTime.add(150 * 60 * 60 * 24, :second)%>
```

Comment:
It's prob ok like this for now, but I think we should convert date values in the backend instead of in the view. And only pass to the view the final values.

Reply:
agree, will create an issue for this